woocommerce/woocommerce
# Relocate WooCommerce Admin Packages

### All Submissions:


### Changes proposed in this Pull Request:

This pull request moves all of the packages in WooCommerce Admin's monorepo at `plugins/woocommerce-admin/packages` into their new home in `packages/js`. It also adds project files so that Nx can interact with it.

### How to test the changes in this Pull Request:

1. Run `pnpm nx build @woocommerce/components` and confirm that it builds the package. This will build all of the other packages in the process.

### Other information:




### Changelog entry

N/A

### FOR PR REVIEWER ONLY:
